app/browser/zoom.js

```diff
@@ -10,11 +17,28 @@ exports = module.exports = () => {
 		}
 
 		if (event.ctrlKey) {
-			if (keyName === '+') {
-				webFrame.setZoomLevel(webFrame.getZoomLevel() + 1);
-			} else if (keyName === '-') {
-				webFrame.setZoomLevel(webFrame.getZoomLevel() - 1);
-			}
+			setNextZoomLevel(keyName, config);
 		}
 	}, false);
-};
\ No newline at end of file
+};
+
+function restoreZoomLevel(config) {
+	ipcRenderer.invoke('getZoomLevel', config.partition).then(zl => {
+		webFrame.setZoomLevel(zl);
+	});
+}
+
+function setNextZoomLevel(keyName, config) {
+	const zoomFactor = zoomLevels[keyName];
+	var zoomLevel = webFrame.getZoomLevel();
+	if (typeof (zoomFactor) !== 'number') {
+		return;
+	}
+
+	zoomLevel = zoomFactor == 0 ? 0 : zoomLevel + zoomFactor;
+	webFrame.setZoomLevel(zoomLevel);
+	ipcRenderer.invoke('saveZoomLevel', {
+		partition: config.partition,
+		zoomLevel: zoomLevel
```

Comment:
Can you change this to webFrame.getZoomLevel?

Just because the setZoomLevel I believe that will ignore out of bonds values.

Reply:
Did you mean zoomLevel property to be set from calling getZoomLevel rather than passing the variable?